VMBackup/main/hostsnapshotter.py

```diff
@@ -162,8 +162,12 @@ def get_snapshot_info(self, responseBody):
             if(responseBody != None):
                 json_reponseBody = json.loads(responseBody)
                 for snapshot_info in json_reponseBody['snapshotInfo']:
-                    blobsnapshotinfo_array.append(HostSnapshotObjects.BlobSnapshotInfo(snapshot_info['isSuccessful'], snapshot_info['snapshotUri'], snapshot_info['errorMessage'], snapshot_info['statusCode']))
-                    self.logger.log("IsSuccessful:{0}, SnapshotUri:{1}, ErrorMessage:{2}, StatusCode:{3}".format(snapshot_info['isSuccessful'], snapshot_info['snapshotUri'], snapshot_info['errorMessage'], snapshot_info['statusCode']))
+                    self.logger.log("IsSuccessful:{0}, SnapshotUri:{1}, ErrorMessage:{2}, StatusCode:{3}, BlobUri: {4}".format(snapshot_info['isSuccessful'], snapshot_info['snapshotUri'], snapshot_info['errorMessage'], snapshot_info['statusCode'], snapshot_info['blobUri']))
+                    if('DDSnapshotIdentifier' in snapshot_info and snapshot_info['DDSnapshotIdentifier']!= None):
+                        blobsnapshotinfo_array.append(HostSnapshotObjects.BlobSnapshotInfo(snapshot_info['isSuccessful'], snapshot_info['snapshotUri'], snapshot_info['errorMessage'], snapshot_info['statusCode'], snapshot_info['blobUri'], snapshot_info['DDSnapshotIdentifier']))
```

Comment:
similar to another place parse DDSnapshotIdentifier in its own class and then send it to BlobSnapshotInfo